mfr/extensions/codepygments/templates/viewer.mako

```diff
@@ -1,8 +1,7 @@
-<link rel="stylesheet" href="${base}/css/default.css">
+<script src="/assets/codepygments.js"></script>
```

Comment:
webpacked script to get the css on the page. This is probably not worth it for this extension, but it would make sense to keep consistency